WebApplication1.sln

```diff
@@ -0,0 +1,16 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebApplication1", "WebApplication1\WebApplication1.csproj", "{D69A989B-5733-4E1D-940D-7E498293CA6B}"
```

Comment:
Let's rename the project&solution to `VerificationWeb` maybe?

Reply:
Renamed